src/content/en/fundamentals/engage-and-retain/web-app-manifest/_redirects.yaml

```diff
@@ -18,3 +18,15 @@ redirects:
 
 - from: /web/fundamentals/engage-and-retain/web-app-manifest/more-information
   to: /web/fundamentals/engage-and-retain/web-app-manifest/#more-information
+
+- from: /web/fundamentals/engage-and-retain/push-notifications/good-notification
+  to: /web/fundamentals/engage-and-retain/push-notifications/display-a-notification
+
+- from: /web/fundamentals/engage-and-retain/push-notifications/handling-messages
+  to: /web/fundamentals/engage-and-retain/push-notifications/push-events
```

Comment:
"Push events" instead of "handling messages"? Many people (including you, I think) have told me to make titles less API-centric and more use case/functionality-centric.